lldb/source/Expression/DWARFExpression.cpp

```diff
@@ -2341,9 +2198,7 @@ bool DWARFExpression::Evaluate(
     // the stack by the called expression may be used as return values by prior
     // agreement between the calling and called expressions.
     case DW_OP_call2:
-      if (error_ptr)
-        error_ptr->SetErrorString("Unimplemented opcode DW_OP_call2.");
-      return false;
+      return llvm::createStringError("Unimplemented opcode DW_OP_call2.");
```

Comment:
drop all the `.` at the end of the error messages

Reply:
and probably drop the uppercase at the start too?